### PR TITLE
add sonata-project/admin-bundle and PHP 7.0 to composer and symfony 3.1 to travis

### PR DESCRIPTION
**Pull request by ickbinhier**

### CODE REVIEW

**Comment by tubssp** [comment]
only sonata-project/admin-bundle master is in the moment supported symfony3

**Comment by PBXg33k** [comment]
Why a (hard) requirement on sonata-admin? Wouldn't a suggest be better for people who don't use Sonata admin?

**Comment by ickbinhier** [comment]
this Bundle used sonata-admin...

https://github.com/lexik/LexikMailerBundle/search?utf8=%E2%9C%93&q=sonata

**Comment by PBXg33k** [comment]
Yeah, for the admin. You can work without the admin, i do it in a pre-existing project which doesn't use Sonata admin (unfortunatly)

**Comment by ickbinhier** [comment]
What is your proposal?

**Comment by PBXg33k** [comment]
Like i said, i'd personally use comper's own suggest block. That allows you to suggest using dependencies with description on what the benefit is. It'll show up on composer install.

**Comment by ickbinhier** [comment]
do you mean that?

**Comment by PBXg33k** [comment]
Yes :)

**Comment by ickbinhier** [comment]
@gnat42 can you please merge this pull request?

**Comment by gnat42** [comment]
@ickbinhier I'm not a project collaborator so nope... ;)

File: composer.json [hunk 1]
```diff
@@ -16,13 +16,16 @@
...
-        "php": ">=5.5.9",
+        "php": ">=5.5.9|~7.0",
         "symfony/framework-bundle": "~2.8|~3.0",
         "symfony/icu": "~1.1",
         "swiftmailer/swiftmailer": ">=4.1.3,~5.0",
         "doctrine/orm": "~2.4",
         "twig/twig": "~1.3"
     },
+    "suggest": {
+        "sonata-project/admin-bundle": "The missing Symfony Admin Generator; require for the frontend"
```
Comment (cedric-g) [inline composer.json]: > require for the frontend

By default Sonata admin is not required for this bundle, we provide a simple bootstrap admin. 
You will need Sonata only if you enable it:
```yml
lexik_mailer:
    sonata_integration: true # false by default
```

**Pull request merged**